labs/haproxy-tech-details.md

```diff
@@ -0,0 +1,146 @@
+<a id="markdown-haproxy-tech-details" name="haproxy-tech-details"></a>
+
+# HAProxy Tech Details
+
+If you are reading this, probably you are curious enough to understand how we configured global ingress for this lab.
+
+Keep in mind the solution used here is far from perfect, and we have used what the environment offered to us in order to
+make it easy for you to get global ingress in the lab.
```

Comment:
Maybe put something in below this that in a production scenario a global load balancer service would be used. This GLB could either be provided by F5, CloudFlare, or other 3rd party solutions.

Reply:
Added.